storybook/src/components/FieldSet/FieldSet.docs.mdx

```diff
@@ -79,6 +93,8 @@ The order of the `id`s is the order in which they are read by screen readers.
 
 <Canvas of={FieldSetStories.CheckboxGroup} />
 
+Note that you should not make a Field Set containing inputs of `type="checkbox"` required.
+
```

Comment:
This is a little confusing. You should set `aria-required` on all Checkboxes in a group that is required. You shouldn't use `required` at all. I'd remove this sentence.

Reply:
Removed